server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -963,40 +964,41 @@ public void run() {
         .numMaxThreads(16).build();
 
     ManagerClientService.Client iface = managerConnection(getManagerAddress());
-    boolean managerDown = false;
+    final AtomicBoolean managerDown = new AtomicBoolean(false);
```

Comment:
I don't think this needs to be an AtomicBoolean, the checking / setting are all happening in the same thread.

Reply:
Yea might not be needed. I think to use `.removeIf()` it needs to be atomic or final so maybe this pattern is not an improvement.

Reply:
You are familiar with the penalty of `volatile` or `Atomic*` variables?

Reply:
I guess in this case with it being used within a future it might make things less parallel since each thread will have to wait to check the variable right